File: app/src/main/java/com/theflexproject/thunder/model/Cast.java
package com.theflexproject.thunder.model;

// Model untuk Cast
public class Cast implements MyMedia {
    private int id;
    private String name;
    private String character;
    private String profilePath;

    public Cast(int id, String name, String character, String profilePath) {
        this.id = id;
        this.name = name;
        this.character = character;
        this.profilePath = profilePath;
    }

    // Getters
    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getCharacter() {
        return character;
    }

    public String getProfilePath() {
        return profilePath;
    }
}

File: app/src/main/java/com/theflexproject/thunder/model/Cast.kt
package com.theflexproject.thunder.model

import android.os.Parcelable
import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import kotlinx.parcelize.Parcelize

@JsonClass(generateAdapter = true)
@Parcelize
data class Cast(
    @Json(name = "id") val id: Int = 0,
    @Json(name = "name") val name: String? = null,
    @Json(name = "character") val character: String? = null,
    @Json(name = "profile_path") val profilePath: String? = null
) : MyMedia, Parcelable
